csm/core/controllers/users.py

```diff
@@ -154,6 +155,11 @@ async def post(self):
         except ValidationError as val_err:
             raise InvalidRequest(f"Invalid request body: {val_err}")
 
+        max_users_allowed = Conf.get(const.CSM_GLOBAL_INDEX, f"{const.CSM_MAX_USERS_ALLOWED}")
+        existing_users_count = await self._service.get_user_count()
+        if existing_users_count>=max_users_allowed:
+            raise CsmPermissionDenied("User creation failed. Maximum user limit reached.")
+
```

Comment:
Minor: create a separate function for the check.

Reply:
@avrnv  done